### ISSUE TITLE
Can't create workspace. 

### ISSUE THREAD

**Issue by jv1000** [Azure/MachineLearningNotebooks#419]
I installed the Azure Machine Learning python SDK and am now trying to test it following the directions here https://docs.microsoft.com/en-us/azure/machine-learning/service/setup-create-workspace

However, I can't create a workspace.  I have an account on Azure ML services, and did everything correctly. The command ws = Workspace.create(.... takes me to a microsoft page, I select the account to use, it then says I've logged in and after 10 seconds I'm taken to the documentation page. According to the video, it is supposed to ask me to select the correct resource. I've tried using the config file and also by providing the parameters to the workspace.create call.  Since this never happens, the workspace is never created and I have to force quit the notebook kernel. It took me days to install the SDK since I don't have admin rights and the portal is blocked by my firm, and I would like some help if possible. thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: f30b9035-3671-a51d-5054-01313f9967ca
* Version Independent ID: 47fb4557-f0fa-6a43-e648-fd4b48921397
* Content: [Install the Azure Machine Learning SDK for Python - Azure Machine Learning Python](https://docs.microsoft.com/en-us/python/api/overview/azure/ml/install?view=azure-ml-py#feedback)
* Content Source: [AzureML/docs-ref-conceptual/install.md](https://github.com/MicrosoftDocs/MachineLearning-Python-pr/blob/live/AzureML/docs-ref-conceptual/install.md)
* Service: **machine-learning**
* Sub-service: **core**
* GitHub Login: @trevorbye
* Microsoft Alias: **trbye**

**Comment by syedsadiqalinaqvi** [comment]
Have you tried creating the workspace manually (instead of programmatically) using Azure Portal?

**Comment by v-strudm-msft** [comment]
Thank you for reaching out to us.  We see this issue was not addressed by us and was posted some time ago. Our apologies. We hope the community response helped. We will close it. Should you still need assistance with this or another issue, please re-post and we will respond promptly.

**Comment by jv1000** [comment]
No I was not able to resolve this. It's because of the interactive login
required, and this is blocked by my company. I wasted a lot of time on this

Thanks though
